### PR TITLE
Add support for flow batches in flow_to_image

### PR DESCRIPTION
**Pull request by NicolasHug**
Follow up to https://github.com/pytorch/vision/pull/5134 and to https://github.com/pytorch/vision/issues/5091, this PR adds support for batches of flows to the `flow_to_image()` util.

CC @oke-aditya

### CODE REVIEW

**Review by datumbox — APPROVED** [review]
LGTM, thanks!

File: torchvision/utils.py [hunk 1]
```diff
@@ -397,42 +397,51 @@ def flow_to_image(flow: torch.Tensor) -> torch.Tensor:
...
     """
 
     if flow.dtype != torch.float:
         raise ValueError(f"Flow should be of dtype torch.float, got {flow.dtype}.")
 
-    if flow.ndim != 3 or flow.size(0) != 2:
-        raise ValueError(f"Input flow should have shape (2, H, W), got {flow.shape}.")
+    orig_shape = flow.shape
+    if flow.ndim == 3:
+        flow = flow[None]  # Add batch dim
```
Comment (oke-aditya) [inline torchvision/utils.py:412]: One small doubt.

Does this modify flow inplace?

Comment (NicolasHug) [reply torchvision/utils.py:412]: No, the 2 objects may be a view on the same underlying memory, but they're different objects


```py
In [1]: import torch
torch.tensor
In [2]: a = torch.tensor([1, 2])

In [3]: b = a[None]

In [4]: b is a
Out[4]: False

In [5]: a, b
Out[5]: (tensor([1, 2]), tensor([[1, 2]]))
```

Comment (oke-aditya) [reply torchvision/utils.py:412]: Thanks for clarification. Will note this.

**Pull request merged**